docs/source/configuration.rst

```diff
@@ -757,14 +757,22 @@ example, only the ``bugzilla`` and ``remind`` plugins are enabled (because
 To detect plugins from extra directories, use the :attr:`~CoreSection.extra`
 option.
 
-Ignore User
------------
+Ignoring Users
+--------------
 
 To ignore users based on their hosts and/or nicks, you can use these options:
 
 * :attr:`~CoreSection.host_blocks`
 * :attr:`~CoreSection.nick_blocks`
 
+Any :ref:`admin <Owner & Admins>` can modify these from IRC with the ``.blocks``
+command. The full syntax is:
+
+.. code-block:: text
+
+    .blocks list [nick|host]
+    .blocks [add|del] [nick|host] pattern
+
```

Comment:
Hmm, this example configuration is invalid. `*@user/AbusiveUser` will crash the bot because the `*` quantifier has nothing to quantify, and the `host_blocks` also don't match on the full hostmask despite the `hostmask` argument name that the `.blocks` command used to take in 7.x.

Nevertheless, I adopted some form of this idea. Thanks for the nudge.